addons/website/views/snippets/s_social_media.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_social_media" name="Social Media">
+    <div class="s_social_media text-left">
+        <h4 class="s_social_media_title d-none">Social Media</h4>
+        <a data-social-id="social_facebook" href="/website/social/facebook" class="s_social_media_facebook o_not_editable" target="_blank">
+            <i class="fa fa-facebook rounded shadow-sm"/>
+        </a>
+        <a data-social-id="social_twitter" href="/website/social/twitter" class="s_social_media_twitter o_not_editable" target="_blank">
+            <i class="fa fa-twitter rounded shadow-sm"/>
+        </a>
+        <a data-social-id="social_linkedin" href="/website/social/linkedin" class="s_social_media_linkedin o_not_editable" target="_blank">
+            <i class="fa fa-linkedin rounded shadow-sm"/>
+        </a>
+        <a data-social-id="social_youtube" href="/website/social/youtube" class="s_social_media_youtube o_not_editable" target="_blank">
+            <i class="fa fa-youtube rounded shadow-sm"/>
+        </a>
+        <a data-social-id="social_instagram" href="/website/social/instagram" class="s_social_media_instagram o_not_editable" target="_blank">
+            <i class="fa fa-instagram rounded shadow-sm"/>
+        </a>
+        <a data-social-id="social_github" href="/website/social/github" class="s_social_media_github o_not_editable" target="_blank">
+            <i class="fa fa-github rounded shadow-sm"/>
+        </a>
+    </div>
+</template>
+
+<template id="s_social_media_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-js="SocialMedia" data-selector=".s_social_media">
+            <we-select string="Title Position" data-apply-to=".s_social_media_title">
+                <we-button data-select-class="d-block">Top</we-button>
+                <we-button data-select-class="">Left</we-button>
+                <we-button data-select-class="d-none">None</we-button>
+            </we-select>
+            <we-select string="Layout" data-apply-to=".fa">
+                <we-button data-select-class="rounded shadow-sm">Square</we-button>
+                <we-button data-select-class="rounded-empty-circle shadow-sm">Circle</we-button>
+                <we-button data-select-class="rounded-circle shadow-sm">Disk</we-button>
+                <we-button data-select-class="m-1">None</we-button>
+            </we-select>
+            <we-select string="Size" data-apply-to=".fa">
+                <we-button data-select-class="">Small</we-button>
+                <we-button data-select-class="fa-2x">Medium</we-button>
+                <we-button data-select-class="fa-3x">Big</we-button>
+            </we-select>
+            <!-- Compatibility, keep reverse logical, don't use `icon_color` -->
+            <we-checkbox string="Color" data-select-class="no_icon_color|"/>
```

Comment:
Duplicated code is bad, we can't obviously do this, would be a pain to maintain.
Just add both selector to data-selector and data-apply-to and